src/pages/playground.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+import Layout from '@theme/Layout';
+import { useLocation } from 'react-router';
+import queryString from 'query-string';
+import { LabelText, Hero, Text } from '@algolia/ui-library';
+import DocSearch from '../components/DocSearch';
+import ErrorBoundary from '../components/ErrorBoundary';
+import algoliasearch from 'algoliasearch';
+import Card from '@algolia/ui-library/public/components/Card';
+
+function Playground() {
+  const {
+    appId: appIdQS = 'BH4D9OD16A',
+    indexName: indexNameQS = '',
+    apiKey: apiKeyQS = '',
+  } = queryString.parse(useLocation().search);
+
+  const [isValidDSCred, setisValidDSCred] = useState(false);
+  const [wrongCredentials, setWrongCredentials] = useState(false);
+  const [appId, setAppId] = useState(appIdQS);
+  const [indexName, setIndexName] = useState(indexNameQS);
+  const [apiKey, setApiKey] = useState(apiKeyQS);
+
+  const fallbackToDocSearchDocCred = () => {
+    setisValidDSCred(false);
+    setAppId('BH4D9OD16A');
+    setIndexName('docsearch');
+    setApiKey('25626fae796133dc1e734c6bcaaeac3c');
+  };
+
+  useEffect(() => {
+    // Credential not provided
+    if (!indexName && !apiKey) {
+      fallbackToDocSearchDocCred();
+      return;
+    }
+    if ((!indexName && !apiKey) || apiKey.length !== 32) {
+      setWrongCredentials(true);
+      fallbackToDocSearchDocCred();
+      return;
+    }
```

Comment:
Do we need to handle this ourselves or can be rely on the `index.search('')` check?

Reply:
Faster check, let's keep it.